Opcodes/py/pythonopcodes.h

```diff
@@ -21,7 +21,13 @@
 #ifndef _pycsound_pythonopcodes_h_
 #define _pycsound_pythonopcodes_h_
 
-#include <Python.h>
+#ifdef _DEBUG
+# undef _DEBUG
+#  include <Python.h>
+# define _DEBUG
+#else
+# include <Python.h>
+#endif
```

Comment:
Exactly the issue I was having but had not spent the time to fix here.  Thanks again!